src/main/java/org/alfresco/repo/security/authority/AuthorityTypeBehaviour.java

```diff
@@ -112,12 +79,38 @@ public void init()
 
     public void onUpdateProperties(NodeRef nodeRef, Map<QName, Serializable> before, Map<QName, Serializable> after)
     {
+        if (modifyingOwnAccount(before, after))
+        {
+            return;
+        }
+
         if(!(AuthenticationUtil.isRunAsUserTheSystemUser() || authorityService.hasAdminAuthority()))
         {
             throw new AccessDeniedException("Only users with ROLE_ADMINISTRATOR are allowed to manage users.");
         }
     }
 
+    private boolean modifyingOwnAccount(Map<QName, Serializable> before, Map<QName, Serializable> after)
+    {
+        String beforeUsername = findUsernameInProperties(before, USERNAME_FIELD, INVALID_USERNAME_VALUE);
+        String afterUsername = findUsernameInProperties(after, USERNAME_FIELD, INVALID_USERNAME_VALUE);
+        if ( afterUsername.equals(beforeUsername) ){

```

Comment:
this is not Alfresco formatting stile for 
{ // it should go on a new line. 
Applies to the entire PR